webapp/channels/src/components/admin_console/custom_enable_disable_guest_accounts_setting.test.tsx

```diff
@@ -18,68 +21,70 @@ describe('components/AdminConsole/CustomEnableDisableGuestAccountsSetting', () =
         showConfirm: false,
     };
 
+    const intlProviderProps = {
+        defaultLocale: 'en',
+        locale: 'en',
+    };
+
+    const renderComponent = (props = {}) => {
```

Comment:
You should be able to use `renderWithContext`, that does most of what you are doing here.